apps/projects/app/components/Content/IssueDetail/components/Detail.js

```diff
@@ -0,0 +1,256 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Badge, Text, theme, ContextMenu, ContextMenuItem } from '@aragon/ui'
+
+import { DropDownButton } from '../../../Shared'
+import githubLogo from '../../../../assets/svg/github.svg'
+
+const GithubLogo = styled.img`
+  height: 1em;
+  vertical-align: middle;
+  margin-right: 5px;
+  filter: invert(59%) sepia(21%) saturate(3955%) hue-rotate(166deg)
+    brightness(96%) contrast(88%);
+`
+
+const StyledTable = styled.div`
+  margin-bottom: 20px;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  border: solid ${theme.contentBorder};
+  border-width: 2px 0;
+  > :not(:first-child) {
+    border-left: 1px solid ${theme.contentBorder};
+    padding-left: 15px;
+  }
+`
+
+const StyledCell = styled.div`
+  padding: 20px 0;
+  align-items: left;
+`
+
+// TODO: shared
+const FieldTitle = styled(Text.Block)`
+  color: ${theme.textSecondary};
+  text-transform: lowercase;
+  font-variant: small-caps;
+  font-weight: bold;
+  margin-bottom: 6px;
+`
+
+const SummaryTable = () => {
+  return (
+    <StyledTable>
+      <StyledCell>
+        <FieldTitle>Experience Level</FieldTitle>
+        <Text color={theme.textPrimary}>Beginner</Text>
```

Comment:
Can we get this handled through a prop and then hardcode the prop instead of hardcoding it directly.

Reply:
Solved at 461c6f3